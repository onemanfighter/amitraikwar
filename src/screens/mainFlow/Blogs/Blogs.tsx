import { Box, Text } from '@chakra-ui/react';

const Blogs = () => {
  return (
    <Box>
      <Text>Blogs</Text>
    </Box>
  );
};

export default Blogs;
