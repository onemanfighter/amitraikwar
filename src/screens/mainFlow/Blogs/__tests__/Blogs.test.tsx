import { render } from '@testing-library/react';
import Blogs from '../Blogs';

describe('Blogs', () => {
  it('should render correctly', () => {
    const { container } = render(<Blogs />);

    expect(container).toMatchSnapshot();
  });
});
