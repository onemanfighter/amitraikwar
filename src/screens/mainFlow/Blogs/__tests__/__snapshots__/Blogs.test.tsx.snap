// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blogs should render correctly 1`] = `
<div>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-0"
    >
      Blogs
    </p>
  </div>
</div>
`;
